src/client/src/app/lib/components/nhsuk-frontend/summary-list.tsx

```diff
@@ -14,7 +14,7 @@ type OnClickActionProps = {
 
 export type SummaryListItem = {
   title: string;
-  value: string;
+  value: string | string[];
```

Comment:
Instead of making this a `string[]` would it be better if we made it `string | ReactNode` similar to how it is done in the `<Table .. />` component?

Reply:
I am not sure I agree. I think in this case doing the simplest thing that works is fine.

Reply:
Yeah fair enough, happy to go with that.